src/cmd/cli/command/commands.go

```diff
@@ -53,6 +53,24 @@ func prettyError(err error) error {
 
 }
 
+func detectComposeUpEndLogEventFunc(service string, host string, eventLog string) bool {
+	result := false
+	if service == "cd" && host == "pulumi" {
+		result = strings.Contains(eventLog, "Update succeeded") ||
+			strings.Contains(eventLog, "Update completed")
+	}
+	return result
+}
+
+func detectComposeDownEndLogEventFunc(service string, host string, eventLog string) bool {
+	result := false
+	if service == "cd" && host == "pulumi" {
+		result = strings.Contains(eventLog, "Destroy succeeded") ||
+			strings.Contains(eventLog, "Update completed")
```

Comment:
change "completed" to "succeeded" in playground's `pulumi.go` to match Pulumi's enum and BYOC